From ba759d2edfcb0e5f754ebfa03a23b2ea53e564ab Mon Sep 17 00:00:00 2001
From: Bart Ribbers <bribbers@disroot.org>
Date: Wed, 9 Jun 2021 12:49:44 +0200
Subject: [PATCH] Autostart pipewire-media-session and pipewire-pulse

Upstream's config does not autostart pipewire-media-session anymore or
pipewire-pulse, which are essential components. It's not easy to start them
externally in a script since it needs the pipewire socket up, and Alpine
doesn't have a mechanism to ensure it (systemd socket activation)

Upon testing it appeared that Pipewire sitll launches fine with this
config when either -media-session or -pulse is missing, so this is safe
to do
---
 src/daemon/pipewire.conf.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/daemon/pipewire.conf.in b/src/daemon/pipewire.conf.in
index 0c0cdd00..c72ae940 100644
--- a/src/daemon/pipewire.conf.in
+++ b/src/daemon/pipewire.conf.in
@@ -234,12 +234,12 @@ context.exec = [
     # but it is better to start it as a systemd service.
     # Run the session manager with -h for options.
     #
-    @comment@{ path = "@session_manager_path@"  args = "@session_manager_args@" }
+    { path = "@session_manager_path@"  args = "@session_manager_args@" }
     #
     # You can optionally start the pulseaudio-server here as well
     # but it is better to start it as a systemd service.
     # It can be interesting to start another daemon here that listens
     # on another address with the -a option (eg. -a tcp:4713).
     #
-    @comment@{ path = "@pipewire_path@" args = "-c pipewire-pulse.conf" }
+    { path = "@pipewire_path@" args = "-c pipewire-pulse.conf" }
 ]
-- 
2.32.0

